subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -405,28 +416,47 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id, uint16_t *handles)
 			cis->lll.rx.ft = cis->lll.tx.ft;
 
 #elif defined(CONFIG_BT_CTLR_CONN_ISO_RELIABILITY_POLICY)
-			/* Utilize Max_Transmission_latency */
-			if (cis->framed) {
-				/* TL = CIG_Sync_Delay + FT x ISO_Interval + SDU_Interval.
-				 * SDU_Interval <= CIG_Sync_Delay
-				 */
-				cis->lll.tx.ft =
-					DIV_ROUND_UP(cig->c_latency - cig->c_sdu_interval -
-							iso_interval_us, iso_interval_us);
-				cis->lll.rx.ft =
-					DIV_ROUND_UP(cig->p_latency - cig->p_sdu_interval -
-							iso_interval_us, iso_interval_us);
-			} else {
-				/* TL = CIG_Sync_Delay + FT x ISO_Interval - SDU_Interval.
-				 * SDU_Interval <= CIG_Sync_Delay
-				 */
-				cis->lll.tx.ft =
-					DIV_ROUND_UP(cig->c_latency + cig->c_sdu_interval -
-							iso_interval_us, iso_interval_us);
-				cis->lll.rx.ft =
-					DIV_ROUND_UP(cig->p_latency + cig->p_sdu_interval -
-							iso_interval_us, iso_interval_us);
+			/* Utilize Max_Transport_latency */
+
+			/* Framed:
+			 *   TL = CIG_Sync_Delay + FT x ISO_Interval + SDU_Interval
+			 *
+			 * Unframed:
+			 *   TL = CIG_Sync_Delay + FT x ISO_Interval - SDU_Interval
+			 *
+			 * Set CIG_Sync_Delay = ISO_Interval as largest possible CIG_Sync_Delay.
+			 * This favors utilizing as much as possible of the Max_Transport_latency,
+			 * and spreads out payloads over multiple CIS events (if necessary).
+			 */
+			uint32_t cig_sync_delay_max = iso_interval_us;
+			int8_t sdu_intv_factor = cis->framed ? 1 : -1;
```

Comment:
So if this can be `-1` then `iso_interval_us + sdu_intv_factor * cig->c_sdu_interval` can be 0, right? So that means that 
```c
tl = cig_sync_delay_max + ft * iso_interval_us + sdu_intv_factor *
										cig->c_sdu_interval;
```

can up being equivalent to `tl = cig_sync_delay_max` - Is that correct? 

If so, could you please add a comment stating the concept of why `sdu_intv_factor = -1` makes sense?

Reply:
Not sure I understand... In some cases, if the ISO_Interval and SDU_Interval are the same - then yes. The factor is there to avoid having the two almost identical statements, times two. With this, there's only one for C_To_P and one for P_To_C.
Should I write something like "Using SDU_Interval sign factor to consolidate code for framed and unframed", or are you looking for something more specific?

Reply:
It may be just me that aren't as familiar with these calculations, but to me it would be more readable to have 2 statements, i.e. 

```c
if (cis->framed) {
    tl = cig_sync_delay_max + ft * iso_interval_us + cig->c_sdu_interval;
} else {
    tl = cig_sync_delay_max + ft * iso_interval_us - cig->c_sdu_interval;
}
```

One such argument is that I now see that my original understanding of the statement was incorrect :D 

But I'll leave it up to you. 

Alternative you could do 
```c
tl = cig_sync_delay_max + ft * iso_interval_us
if (cis->framed) {
    tl += cig->c_sdu_interval;
} else {
    tl -= cig->c_sdu_interval;
}
```

if you want to avoid duplicated code.